docs/UserGuide.md

```diff
@@ -157,8 +160,7 @@ Format: `add n/FOOD_NAME c/CALORIE t/MEAL_TYPE`
 * Each field can only be specified once.
 * `MEAL_TYPE` can only take on three values: breakfast, lunch or dinner.
 * `CALORIE` can only take in a non-negative integer that is less than 2147483648
-* `FOOD_NAME` should only contain alphanumeric characters and spaces, and should not be blank. Names should also not 
-exceed 27 characters.
+* `FOOD_NAME` should only contain alphanumeric characters and spaces, and should not be blank. Names should also not exceed 27 characters.
```

Comment:
To be consistent with their order in the command format, would it be better to re-order this so that the bullet point for `FOOD_NAME` comes first, followed by `CALORIE` then `MEAL_TYPE`? (I think this also appear in `edit` as well)

Also, there seems to be a missing full stop for line 162!

Reply:
Fixed!